types/klaw-sync/klaw-sync-tests.ts

```diff
@@ -1,23 +1,24 @@
-import * as path from 'path'
 import klawSync = require('klaw-sync')
+import * as fs from 'fs'
 
 const outputMessage = (result: klawSync.Item) => {
     console.log(`file: ${result.path} has size '${result.stats.size}'`)
 }
 
 klawSync('/some/dir').forEach(outputMessage)
 
-const defaultOptions = {}
+const defaultOptions: klawSync.Options = {}
```

Comment:
You might be able to avoid declaring `klawSync.Options` here and below. Did you encounter a specific error?

Reply:
I actually added the explicit type to be more consistent with the change for the `const options` below.

I wanted to make sure the test isn't passing non-`Options` keys.

Before I added the explicit type, nothing ensured `noRecurseOnFailedFilter` is not a valid `Options` key, as the type of `options` was inferred and compatible with the call to `klawSync()`.

This isn't a must, but gives a bit of added value IMO. Would you like me to revert this back?

Reply:
Aha, I missed that detail in the test below. All good!